geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreStatsTest.scala

```diff
@@ -297,6 +297,12 @@ class AccumuloDataStoreStatsTest extends Specification with TestWithMultipleSfts
         }
       }
 
+      "handle empty queries with exact stats" >> {
+        val filter = "dtg > '2019-01-01T00:00:00.000Z' AND dtg < '2019-01-02T00:00:00.000Z' AND dtg > currentDate('-P1D')"
```

Comment:
Brittle unit test!  You never know who may set their system back to beginning of 2019 and try to build GeoMesa!

Bwhahahhahaha!